EOFTests/eof_validation/EOF1_embedded_container.json

```diff
@@ -0,0 +1,55 @@
+{
+  "EOF1_embedded_container": {
+    "vectors": {
+      "EOF1_embedded_container_0": {
+        "code": "0xef00010100040200010006030001001404000000008000016000e0000000ef000101000402000100010400000000800000fe",
+        "results": {
+          "Prague": {
+            "result": true
+          }
+        }
+      },
+      "EOF1_embedded_container_1": {
+        "code": "0xef00010100040200010006030001001404000200008000016000e0000000ef000101000402000100010400000000800000fe",
```

Comment:
The top level container is missing 2 data bytes.  Same issue I ahd with a prior test, do these represent deployed contracts?  i.e. the ones EOFCREATE / TXCREATE would see?

Reply:
Given that top level EOF will need to be validated with truncated data, this seems to be a valid test then.  What we will need are execution tests where return data has truncated, exact, and excess data for both EOFCREATE and TXCREATE.

Reply:
> Given that top level EOF will need to be validated with truncated data, this seems to be a valid test then. What we will need are execution tests where return data has truncated, exact, and excess data for both EOFCREATE and TXCREATE.

I see this comment predates the one you've posted about the disconnect. Has it become obsolete now? If not, I'm not sure what kind of tests you're proposing, what do you mean by "return data" in this context?